Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -136,7 +149,7 @@ private extension AppSettingsStore {
 
     func loadTrackingProvider(siteID: Int,
                               onCompletion: (ShipmentTrackingProvider?, ShipmentTrackingProviderGroup?, Error?) -> Void) {
-        guard let allSavedProviders = read(from: selectedProvidersURL) else {
+        guard let allSavedProviders = readPList(from: selectedProvidersURL) as [PreselectedProvider]? else {
```

Comment:
I know we are reading and writing from plist files, but that sounds like an implementation detail to me, that's why I left it out of the method name. I don't have any strings feelings about it anyway.

Reply:
Yea makes sense, renamed in: https://github.com/woocommerce/woocommerce-ios/pull/1242/commits/05be87235355881c97d9619ec299481729564219